vhd/lib/libvhd.c

```diff
@@ -2876,7 +2876,7 @@ vhd_initialize_footer(vhd_context_t *ctx, int type, uint64_t size)
 	ctx->footer.type         = type;
 	ctx->footer.saved        = 0;
 	ctx->footer.data_offset  = 0xFFFFFFFFFFFFFFFFULL;
-	strcpy(ctx->footer.crtr_app, "tap");
+	strncpy(ctx->footer.crtr_app, "tap", sizeof(ctx->footer.crtr_app));
```

Comment:
Why is this necessary for Coverity here? It is trivial to see that an overrun can not happen.